src/main/java/sorald/cli/RepairCommand.java

```diff
@@ -55,7 +56,7 @@
                 names = {Constants.ARG_RULE_KEY},
                 description =
                         "Choose one of the following rule keys:\n"
-                                + Processors.RULE_DESCRIPTIONS
+                                + SonarProcessorRepository.RULE_DESCRIPTIONS
```

Comment:
We would also need to generalise `RepairCommand` using the service loader. However, it is not in the scope of the current PR. Putting this here to keep a mental note.